### ISSUE TITLE
Feature Suggestion: Suggestions for CSS Next Steps.

### ISSUE THREAD

**Issue by FinVamp1** [ShekharGupta1988/AppLensV2#159]
Sometimes it's not clear even from all the detectors what the root causes are for a customer incident. Was discussing with Shekhar how it might be good to have some recommendations for how CSS could proceed if AppLensv2 doesn't give them the answer.

E.G 

High CPU Usage consistently from W3WP.exe -> Azure Service Profiler or Application Insights.
High Time Taken -> Auto-Heal and DaaS actions.

**Comment by ShekharGupta1988** [comment]
Will discuss this. Right now we show solutions which are really mitigation steps for customers in case of specific problem.
Its a good suggestion to add "CSS Next Steps", where we would need support engineers contribution to our document repo.

**Comment by PraveenTB** [comment]
SIA is already exposing solutions. The solutions can be of 2 types - "Quick Mitigation actions", "Deep troubleshooting options". I believe the ask is of the 2nd type. and the Deep troublshooting options such be such that its valid for both customers and CSS. It will be even nicer if CSS can copy these and email it to customers.

**Comment by stevenernst131** [comment]
Is this a backend ask or a UI ask? We are adding deep troubleshooting options on the back end now, and we should keep adding more. However we will need to expose in the UI so we can start seeing the effect. However I think Shekhar owns that work item since he is doing something for the now solutions as well

**Comment by PraveenTB** [comment]
Yes this is the ask on backend and expose to UI. IMO, shekhar's workitem to expose solutions would cover this item as well
